recompose-idea/src/main/kotlin/recompose/plugin/copypaste/RecomposeCopyPasteProcessor.kt

```diff
@@ -58,14 +59,28 @@ class RecomposeCopyPasteProcessor : CopyPastePostProcessor<TextBlockTransferable
     override fun extractTransferableData(
         content: Transferable
     ): List<TextBlockTransferableData> {
-        if (content.isDataFlavorSupported(CopiedXMLCode.DATA_FLAVOR)) {
-            // There's some matching data in this paste, let's return it to process it.
-            return listOf(
-                content.getTransferData(CopiedXMLCode.DATA_FLAVOR) as TextBlockTransferableData
-            )
-        }
 
-        return emptyList()
+        return if (content.isDataFlavorSupported(DataFlavor.stringFlavor)) {
+            val text = content.getTransferData(DataFlavor.stringFlavor) as String
+            val xmlPreamble = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
```

Comment:
I think that is a good start. Eventually we probably want to support incomplete snippets too. But that's also #41 and #5. I guess we can just file a follow-up issue and then see what a good strategy would be. Maybe looking for the start of known elements.